### ISSUE TITLE
CheckEventLog information severity

### ISSUE THREAD

**Issue by koga5** [mickem/nscp#93]
I tried "severity NOT IN" style like in the samples, and also tried with level, but the result the same as below.
There are more "information" lines in the previous 50 minutes, and those filtered out well, but this not.
nscp 0.4.3.88
```
CheckEventLog file=application file=system MaxWarn=1 MaxCrit=10 "filter=generated gt -50m AND severity ne 'information'" truncate=800 unique descriptions "syntax=%severity%: %source%: %message%"
D   eventlog Created command: warn=count>=1 crit=count>=10 file=application file=system unique filter=written gt -50m AND level ne 'information' detail-syntax=${level}: ${source}: ${message} top-syntax=${list}
L        cli WARNING: information: Microsoft-Windows-Security-SPP: The SoftwareProtection service has started. 6.1.7601.17514, information: Microsoft-Windows-Security-SPP: The Software Protection service has stopped.
L        cli  Performance data: 'count'=2;1;10
```

**Comment by mickem** [comment]
Severity is *NOT* what you expect, always use level instead.

**Comment by koga5** [comment]
Doesn't matter what I use. 
Maybe the problem is whit the "Microsoft-Windows-Security-SPP"? 
Any other suggestion? 

CheckEventLog file=application file=system MaxWarn=1 MaxCrit=10 "filter=generated gt -1d AND level NOT IN ('error','information')" truncate=800 unique descriptions "syntax=%severity%: %source%: %message%"
D   eventlog Created command: warn=count>=1 crit=count>=10 file=application file=system unique filter=written gt -1d AND level NOT IN ('error','information') detail-syntax=${level}: ${source}: ${message} top-syntax=${list}
L        cli CRITICAL: information: Microsoft-Windows-Security-SPP: The Software Protection service has started. 6.1.7601.17514, information: Microsoft-Windows-Security-SPP: The Software Protection service has stopped. , warning: VMware Tools: [ warning] [vmusr:vmtoolsd] Failed registration of app type 2 (Signals) from plugin unity.
L        cli  Performance data: 'count'=19;1;10

CheckEventLog file=application file=system MaxWarn=1 MaxCrit=10 "filter=generated gt -1d AND level NOT IN ('error','information','warning')" truncate=800 unique descriptions "syntax=%severity%: %source%: %message%"
D   eventlog Created command: warn=count>=1 crit=count>=10 file=application file=system unique filter=written gt -1d AND level NOT IN ('error','information','warning') detail-syntax=${level}: ${source}: ${message} top-syntax=${list}
L        cli CRITICAL: information: Microsoft-Windows-Security-SPP: The Software Protection service has started. 6.1.7601.17514, information: Microsoft-Windows-Security-SPP: The Software Protection service has stopped.
L        cli  Performance data: 'count'=17;1;10

**Comment by mickem** [comment]
If you run:
```
CheckEventLog file=application file=system MaxWarn=1 MaxCrit=10 "filter=generated gt -1d AND level NOT IN ('error','information','warning')" truncate=800 unique descriptions "syntax=%level%"
```

Please post the output of that command...

**Comment by koga5** [comment]
If you look to the line begining with "D", what is the interpretation of the command, severity turns into level. So basically I ran the exact same command. Despite of it, I tried what you suggested, and it worked as expected, but my original command is also working as expected now. (Of course I tried to increase the number of days). So I still think that it's a bug, but now I can't reproduce it anymore.
Thanks for trying to help me :)

**Comment by mickem** [comment]
Yes, you are indeed correct...

Interesting...
Apparently the meaning of Levels is textual (ie. the value is mapped to a string) which can be customized (in the new API).

I might fix this in 0.4.3 but probably not soon as it is a bit complicated to resolve.

**Comment by koga5** [comment]
I can reproduce it on an another machine now :)

CheckEventLog file=application file=system MaxWarn=1 MaxCrit=1 "filter=generated gt -10m AND level NOT IN ('success', 'informational') and source not in ('Microsoft-Windows-TerminalServices-Printers')" truncate=800 unique descriptions "syntax=%severity%: %source%: %message%"

D   eventlog Created command: warn=count>=1 crit=count>=1 file=application file=system unique filter=written gt -10m AND level NOT IN ('success', 'informational') and source not in ('Microsoft-Windows-TerminalServices-Printers') detail-syntax=${level}: ${source}: ${message} top-syntax=${list}

L        cli CRITICAL: information: Microsoft-Windows-Security-SPP: A szoftvervédelmi szolgáltatás elindult. 6.1.7600.16385, warning: VMware Tools: [ warning] [vmusr:vmtoolsd] Failed registration of app type 2 (Signals) from plugin unity.

L        cli  Performance data: 'count'=2;1;1

The hungarian characters in the log message can couse problems?

**Comment by mickem** [comment]
This is fixed in 0.5.0.17
